airflow/providers/postgres/CHANGELOG.rst

```diff
@@ -24,6 +24,18 @@
 Changelog
 ---------
 
+5.2.2
+.....
+
+Misc
+~~~~
+
+* ``Add common-sql lower bound for common-sql (#25789)``
+
+.. Review and move the new changes to one of the sections above:
+   * ``Rename schema to database in 'PostgresHook' (#26436)``
```

Comment:
Why it's not in the above section?

Reply:
NVM the PR is being reverted https://github.com/apache/airflow/pull/26734#issue-1388464046